cypress/integration/unit/settings/general/shouldTestSettingsOverviewPage.spec.js

```diff
@@ -74,4 +74,15 @@ context('Settings overview page tests', () => {
     cy.openSettingsModal(manageSignaturesSelector);
     cy.closeSettingsModal();
   });
+
+  it('Upload a users CSV', () => {
+    cy.route('/');
+    cy.get(settingsSelector).click();
+    cy.url().should('include','instellingen/overzicht');
+    cy.get('.vlc-toolbar__left').contains('Gebruikersbeheer').click();
```

Comment:
Ik zou hier gebruik maken van selectors i.p.v. de css klassen. We zijn de codebase momenteel aan het moven naar het werken van met de data-selectors.